packages/panzer/adapters-stk/src/Panzer_STK_LocalMeshUtilities.cpp

```diff
@@ -326,7 +326,7 @@ buildGhostedCellOneRing(const Teuchos::RCP<const Teuchos::Comm<int> > & comm,
 {
 
   PANZER_FUNC_TIME_MONITOR("panzer_stk::buildGhostedCellOneRing");
-  typedef Tpetra::CrsMatrix<int,int,GO> crs_type;
+  typedef Tpetra::CrsMatrix<int,int,GO,panzer::TpetraNodeType> crs_type;
```

Comment:
In some places, you template on LO, but here, you use `LO=int`.  We haven't tested with `LO != int` for years, but you may want to fix this inconsistency at some point (not necessarily now).

Reply:
yeah - we were supporting LO as a template parameter back in 2015/2016. Then about 2 years ago we decided LO should always be int in panzer. So newer code just uses int directly  and older code still has LO as a template parameter. We should fix this, just hasn't been high on the priority list.